src/core/services/theming/theming.js

```diff
@@ -1003,6 +1000,38 @@ function generateAllThemes($injector, $mdTheming) {
       }
     });
   }
+
+  // Split a string representing a CSS file producing an array with a rule in each element of it.
+  function splitCss(themeCss) {
+    var result = [];
+    var currentRule = '';
+    var openedCurlies = 0;
+    var closedCurlies = 0;
+
+    for(var i=0; i < themeCss.length;i++){
```

Comment:
Could you add spaces between the expressions? e.g. `for(var i=0; i < themeCss.length;i++){` -> `for (var i = 0; i <  themeCss.length ; i++) {`